protobuf/flow/executiondata/executiondata.proto

```diff
@@ -191,3 +195,12 @@ message EventFilter {
   // a mainnet node. Addresses may optionally include the 0x prefix.
   repeated string address = 3;
 }
+
+message GetRegisterValuesRequest {
+  uint64 block_height = 1;
+  repeated bytes register_ids = 2;
+}
+
+message GetRegisterValuesResponse {
+  repeated bytes values = 1;
```

Comment:
please add comments similar to the other messages/arguments in this file. These are pretty straight forward so you don't need a ton